lib/feature_flag.rb

```diff
@@ -0,0 +1,15 @@
+class FeatureFlag
```

Comment:
🗒  Very dump feature flag management. Ideally we would be able to persist the features and their state in the database, so that we can switch them on demand (as an admin user). For now, until such requirement is raised, kept it as simple as possible.